### PR TITLE
Avoid 401 Unauthorized exception when working with the `2-Call-OwnApi` use case

### PR DESCRIPTION
**Pull request by balcsok**
## Purpose

Set Web API manifest key `accessTokenAcceptedVersion` correctly to avoid 401 Unauthorized exception when working with the `2-Call-OwnApi` use case.

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[x] Bugfix
[ ] Feature
[x] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Comment by balcsok** [comment]
@microsoft-github-policy-service agree

**Comment by balcsok** [comment]
#193

**Review by jmprieur — APPROVED** [review]
Lgtm
Thanks @balcsok

File: 2-Call-OwnApi/README.md [hunk 1]
```diff
@@ -433,6 +435,12 @@ It's also possible to get certificates from an [Azure Key Vault](https://docs.mi
 
 Build and run your project. You have the same output, but this time, your application is authenticated with Microsoft Entra ID with the certificate instead of the application secret.
 
+## Getting HTTP 401 Unauthorized in the `daemon-console`
+
+This might be caused by an incorrectly set key in the manifest of the `TodoList-webapi-daemon-v2` [App registrations](https://go.microsoft.com/fwlink/?linkid=2083908) in your Microsoft Entra tenant.
+
+Set `accessTokenAcceptedVersion` to `2`, see [Register the service app (TodoList-webapi-daemon-v2)](#register-the-service-app-todolist-webapi-daemon-v2) and the [Microsoft Docs for `accessTokenAcceptedVersion`](https://learn.microsoft.com/en-us/entra/identity-platform/reference-app-manifest#accesstokenacceptedversion-attribute).
```
Comment (jmprieur) [inline 2-Call-OwnApi/README.md:442]:
```suggestion
Set `accessTokenAcceptedVersion` to `2`, see [Register the service app (TodoList-webapi-daemon-v2)](#register-the-service-app-todolist-webapi-daemon-v2) and the [Microsoft Docs for `accessTokenAcceptedVersion`](https://learn.microsoft.com/entra/identity-platform/reference-app-manifest#accesstokenacceptedversion-attribute).
```

**Pull request merged**